File: app/src/main/java/com/sharpdroid/registroelettronico/adapters/Holders/AbsencesHolder.java
package com.sharpdroid.registroelettronico.adapters.Holders;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.sharpdroid.registroelettronico.R;

import butterknife.BindView;
import butterknife.ButterKnife;
import de.hdodenhof.circleimageview.CircleImageView;


public class AbsencesHolder extends RecyclerView.ViewHolder {
    @BindView(R.id.date)
    public TextView date;
    @BindView(R.id.hour)
    public TextView hour;
    @BindView(R.id.type)
    public TextView type_text;
    @BindView(R.id.done)
    public ImageView done;
    @BindView(R.id.type_color)
    public CircleImageView type_color;
    @BindView(R.id.layout)
    public View layout;

    public AbsencesHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/com/sharpdroid/registroelettronico/adapters/holders/AbsencesHolder.kt
package com.sharpdroid.registroelettronico.adapters.holders

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import android.widget.TextView
import de.hdodenhof.circleimageview.CircleImageView
import kotlinx.android.synthetic.main.adapter_absence.view.*


class AbsencesHolder(l: View) : RecyclerView.ViewHolder(l) {
    var date: TextView = l.date
    var hour: TextView = l.hour
    var type_text: TextView = l.type
    var done: ImageView = l.done
    var type_color: CircleImageView = l.type_color
    var layout: View = l.layout
}